aspnetcore/host-and-deploy/linux-nginx.md

```diff
@@ -393,10 +483,24 @@ Add the */etc/nginx/proxy.conf* configuration file:
 
 [!code-nginx[](linux-nginx/proxy.conf)]
 
+# [Ubuntu](#tab/linux-ubuntu)
+
 **Replace** the contents of the */etc/nginx/nginx.conf* configuration file with the following file. The example contains both `http` and `server` sections in one configuration file.
 
 [!code-nginx[](linux-nginx/nginx.conf)]
 
+# [Red Hat Enterprise Linux](#tab/linux-rhel)
+
+Modify `/etc/nginx/nginx.conf`. Open it in a text editor, and replace the `http{}` and `server{}` code blocks with contents with the following snippet:
+
+# [SUSE Linux Enterprise Server](#tab/linux-sles)
+
+Modify `/etc/nginx/nginx.conf`. Open it in a text editor, and replace the `http{}` and `server{}` code blocks with contents with the following snippet:
+
+---
+
+[!code-nginx[](linux-nginx/nginx.conf)]
```

Comment:
I'm not sure why these are includes instead of snippets?

Reply:
I don't know the original author's intent.  Showing it is a .conf example in its entirety available perhaps?